core/drivers/crypto/se050/glue/include/i2c_native.h

```diff
@@ -9,8 +9,24 @@
 
 #include <kernel/rpc_io_i2c.h>
 
+#if defined(CFG_CORE_SE05X_I2C_TRAMPOLINE) && CFG_CORE_SE05X_I2C_TRAMPOLINE
+static inline TEE_Result native_i2c_transfer(
+				struct rpc_i2c_request *req __unused,
+				size_t *bytes __unused)
```

Comment:
for consistency with other header and source files, prefer line escape as below:
```c
static inline TEE_Result
native_i2c_transfer(struct rpc_i2c_request *req __unused,
		    size_t *bytes __unused)

Reply:
done